src/redux/preview.js

```diff
@@ -763,10 +763,11 @@ module.exports.getRemixes = id => (dispatch => {
     });
 });
 
-module.exports.getProjectStudios = id => (dispatch => {
+module.exports.getProjectStudios = (id, ownerUsername, isAdmin, token) => (dispatch => {
     dispatch(module.exports.setFetchStatus('projectStudios', module.exports.Status.FETCHING));
     api({
-        uri: `/projects/${id}/studios`
+        uri: `${isAdmin ? '/admin' : `/users/${ownerUsername}`}/projects/${id}/studios`,
+        authentication: token
```

Comment:
In other places that we do this, we specifically check for falsey token and set it to `null`, I think we should do the same here (see e.g. https://github.com/LLK/scratch-www/blob/5899420a27a13311ebff7e2c6f577df947f704c6/src/redux/preview.js#L469)

Reply:
We want to use the token even if the user is not an admin, but I think you're still right that we should be explicitly using `null`. Will change.

Reply:
OK, I used an approach that's used elsewhere in preview.js, where if token is truthy, I add it to the opts using `Object.assign()`.